src/main/scala/firrtl/stage/FirrtlAnnotations.scala

```diff
@@ -0,0 +1,304 @@
+// See LICENSE for license details
+
+package firrtl.stage
+
+import firrtl._
+
+import firrtl.ir.Circuit
+import firrtl.annotations.NoTargetAnnotation
+import firrtl.transforms.BlackBoxTargetDirAnno
+import firrtl.options.HasScoptOptions
+import logger.LogLevel
+import scopt.OptionParser
+
+/** Indicates that a subclass is an [[annotations.Annotation]] with an
+  * option consummable by [[HasFirrtlExecutionOptions]]
```

Comment:
I think HasFirrtlExecutionOptions is no longer relevant

Reply:
You're right.